engine/runtime/src/main/java/org/enso/ClassLoaderConstants.java

```diff
@@ -13,9 +13,9 @@ public class ClassLoaderConstants {
    * consistent.
    */
   public static final List<String> CLASS_DELEGATION_PATTERNS =
-      List.of("org.graalvm", "java", "org.slf4j", "ch.qos");
+      List.of("org.graalvm", "java", "org.slf4j", "ch.qos", "io.helidon");
```

Comment:
Good. This ensures that our boot class loader deleagates loading of all `io.helidon.*` classes to the system class loader, which picks the classes from JARs inside the `component` directory.